src/core/agent/services/multiSigService.ts

```diff
@@ -1115,11 +1115,13 @@ class MultiSigService extends AgentService {
       .filter((signing: any) => signing.aid !== ourIdentifier.id)
       .map((member: any) => member.aid);
 
-    if (admitExnToGrant) {
-      const [, ked] = Saider.saidify(admitExnToGrant);
+    if (grantToJoin) {
+      const { grantExn, atc } = grantToJoin;
+      const previousAtc = atc.exn;
```

Comment:
I think lets call this `receivedAtc` because otherwise it feels like it's from a different "previous" exn message but it's the same message being re-signed